README.md

```diff
@@ -28,21 +29,10 @@ Find `BUILD.bat` here in the root folder of tgstation, and double click it to in
 
 After it finishes, you can then [setup the server](.github/RUNNING_A_SERVER.md) normally by opening `tgstation.dmb` in DreamDaemon.
 
-**Building tgstation in DreamMaker directly is now deprecated and might produce errors**, such as `'tgui.bundle.js': cannot find file`.
+**Building Merchant Station in DreamMaker directly is now deprecated and might produce errors**, such as `'tgui.bundle.js': cannot find file`.
 
 **[How to compile in VSCode and other build options](tools/build/README.md).**
 
-## Requirements for contributors
-[Guidelines for Contributors](.github/CONTRIBUTING.md)
-
-[/tg/station HACKMD account](https://hackmd.io/@tgstation) - Design documentation here
```

Comment:
we have a hackmd, you should include it here.

Reply:
> we have a hackmd, you should include it here.

I was looking at our hackmd and thinking of including it in the rules section instead (since it's only that), but I guess I forgot about it (was on 3h sleep lmao)